graphics/cairo-devel/Portfile

```diff
@@ -114,6 +114,11 @@ platform macosx {
     variant_set             quartz
 }
 
+if {[string match *gcc-4.* ${configure.compiler}]} {
+    # gcc-4 on OS X 10.5 defaults to gnu89 which is "ISO C90 plus GNU extensions". We need gnu99.
```

Comment:
Could we remove 10.5 here? It is compiler-specific, not OS-specific. Otherwise include a range of 10.4–10.6, if wanted. Sorry for bothering with this again, I am just afraid that misleading comment may result in someone later on deciding to change condition to `${os.major} == 9`.

Reply:
Sure. No objections here. I've updated my pr.